<launch>
    <node name="morphological_segmentation" pkg="filters" type="morphological_segmentation" output="screen">
        <param name="subscribed_topic" type="string" value="/cloud_filtered"/>
        <param name="published_topic" type="string" value="cloud_without_ground"/>
        <param name="maxWindowSize" type="int" value="20"/>
        <param name="slope" type="double" value="1.0"/>
        <param name="initialDistance" type="double" value="0.5"/>
        <param name="maxDistance" type="double" value="3.0"/>
    </node>
</launch>
